src/lightning/pytorch/plugins/__init__.py

```diff
@@ -22,7 +21,6 @@
     "XLACheckpointIO",
     "DeepSpeedPrecisionPlugin",
     "DoublePrecisionPlugin",
-    "IPUPrecisionPlugin",
```

Comment:
These classes need to be added to the graveyard too, right?

Reply:
Added to the graveyard, but shall we keep the reference also here?